node_package/src/clientStartup.js

```diff
@@ -63,6 +63,10 @@ function turbolinksVersion5() {
   return (typeof Turbolinks.controller !== 'undefined');
 }
 
+function turbolinksSupported() {
+  return Turbolinks.supported;
```

Comment:
1. Is this compatible with v2 and v5 turbolinks?
2. What browsers are not supported?

@ka2n